packages/fabric8-tenant-che-mt/src/main/fabric8/che-exec-sa.yml

```diff
@@ -0,0 +1,3 @@
+metadata:
+  name: che-exec
+automountServiceAccountToken: true
```

Comment:
According to Mario's [comment](https://github.com/eclipse/che/issues/10991#issuecomment-418813072) we should not use this SA in pods other than Che workspaces. Do you think that we need to remove automount when setting a particular SA would be supported in upstream Che?